### PR TITLE
Release v0.0.1a8

### PR DESCRIPTION
**Pull request by kairoaraujo**
- Bump version
- Update the Copyright

Signed-off-by: Kairo de Araujo <kdearaujo@vmware.com>

### CODE REVIEW

File: repository_service_tuf/__version__.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# SPDX-FileCopyrightText: 2022 VMware Inc
+# SPDX-FileCopyrightText: 2022-2023 VMware Inc
 #
 # SPDX-License-Identifier: MIT
 
-version = "0.0.1a7"
+version = "0.0.1a8"
 copyright = "Copyright (c) 2022 VMware Inc"
```
Comment (MVrachev) [inline repository_service_tuf/__version__.py:6]: Can you also update this year?

**Comment by kairoaraujo** [comment]
@MVrachev, #191 is required because it fixes the test dependencies.

**Comment by kairoaraujo** [comment]
Rebased

**Comment by kairoaraujo** [comment]
@MVrachev, ready for another review. Thank you.

**Review by MVrachev — APPROVED** [review]

**Pull request merged**